pkg/common/format.go

```diff
@@ -12,6 +13,10 @@ var (
 )
 
 func FormatYAML(data []byte) ([]byte, error) {
+
+	// Unescape HTML entities first
+	data = []byte(html.UnescapeString(string(data)))
```

Comment:
Why this is needed here?

Reply:
The double quotes " were being converted to html entity code &#34. so to parse that as it is in annotations I added it.